### PR TITLE
Update readthedocs config from package template

### PR DESCRIPTION
**Pull request by lpsinger**

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 version: 2
 
 build:
-  image: latest
+  os: ubuntu-20.04
```
Comment (pllim) [inline .readthedocs.yml:4]:
```suggestion
  os: ubuntu-22.04
```

File: .readthedocs.yml [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+  tools:
+    python: 3.8
```
Comment (pllim) [inline .readthedocs.yml:6]:
```suggestion
    python: "3.11"
```

File: .readthedocs.yml [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
 
 python:
```
Comment (pllim) [inline .readthedocs.yml:8]:
```suggestion
python:
  system_packages: false
```

**Review by pllim — COMMENTED** [review]
Might as well upgrade it?

**Comment by pllim** [comment]
Hmm I don't see RTD being build in this PR. So I cannot see if this works or not.

**Comment by lpsinger** [comment]
> Hmm I don't see RTD being build in this PR. So I cannot see if this works or not.

Merge it? YOLO!

**Comment by pllim** [comment]
Or you can enable https://docs.readthedocs.io/en/stable/pull-requests.html ? 😸

**Comment by lpsinger** [comment]
> Or you can enable https://docs.readthedocs.io/en/stable/pull-requests.html ? 😸

Thanks! I closed and reopened this PR, which triggered a Readthedocs build. https://readthedocs.org/projects/astropy-healpix/builds/21428978/

**Comment by lpsinger** [comment]
Looks like that readthedocs build worked.

**Review by pllim — APPROVED** [review]
Great. Thanks!

**Comment by pllim** [comment]
I used "Squash and Merge". Hope that's okay with you!

**Comment by lpsinger** [comment]
> I used "Squash and Merge". Hope that's okay with you!

🪰 🔨

**Pull request merged**